example-dotnet/Program.cs

```diff
@@ -11,8 +22,107 @@ static void Main(string[] args)
             Say.hello("example");
             Console.WriteLine("Hello World!");
 
+	    // Usage of BCrypt.Net API
+
+            // The following API call is meant to demonstrate
+            // vulnerable methods (the simplest case).
+              
             var hashedPassword = BCrypt.Net.BCrypt.HashPassword("my-secret-password");
             BCrypt.Net.BCrypt.Verify("wrong-password", hashedPassword);
+
+            // The next API calls to four libraries are meant to
+            // demonstrate update advisor and auto pull request
+            // features.
+            //
+            // The first two are the cases where the update versions
+            // are of the same frameworks as the currently-used
+            // version.
+            // +--------------------+--------------+----------------+-----------+-----------------+----------+
+            // | Dependency         | Current      | Current        | Update    | Update          | Breaking |
+            // |                    | Version      | Framework(s)   | Version   | Framework(s)    | Update   |
+            // +====================+==============+================+===========+=================+==========+
+            // | recurly-client-api | 1.8.0        | net35          | 1.8.1 or  | net35           | No       |
+            // |                    |              |                | 1.11.4    |                 |          |

```

Comment:
Is 1.11.4 meaningful? If not we can leave it out and have just one version as the target to simplify examples. 1.8.1 would be picked based on our current heuristics.

Reply:
Actually recurly-client-api has two vulnerabilites, one is fixed in 1.11.4, and another in 1.8.1. I removed the mention of 1.8.1.

Reply:
I've updated the QA database and 1.11.4 is the recommended update now.